documentation/assemblies/deploying/assembly-deploy-verify.adoc

```diff
@@ -3,11 +3,18 @@
 // deploying/master.adoc
 
 [id="deploy-verify_{context}"]
-= Verifying the Strimzi deployment
+= Setting up client access to the Kafka cluster
 
-Having xref:deploy-tasks_{context}[deployed Strimzi], the procedure in this section shows how to deploy example producer and consumer clients.
+Having xref:deploy-tasks_{context}[deployed Strimzi], the procedures in this section show how to:
 
-The procedure assumes a Strimzi is available and running in a Kubernetes cluster.
+* Deploy example producer and consumer clients, which you can use to verify your deployment
+* How to set up external client access to the Kafka cluster
++
+The steps to set up access to the Kafka cluster for a client outside Kubernetes are a lot more involved,
```

Comment:
Not sure "involved" is friendly for localization. Maybe we can say "more complex" instead. 

"The steps to set up access to the Kafka cluster for a client outside Kubernetes are more involved..."

Reply:
Good point. I've changed.